Web/starter_project_vanguard/pages/profile/index.tsx

```diff
@@ -0,0 +1,62 @@
+import Link from 'next/link'
+import React, { useState } from 'react'
+
+import AccountSetting from '@/components/profile/AccountSetting'
+import PersonalInfo from '@/components/profile/PersonalInfo'
+
+const Profile: React.FC = () => {
+  const [tab, setTab] = useState<number>(0)
+  const handleTabChange = (tab: number) => {
+    setTab(tab)
+  }
+
+  return (
+    <div className="px-10">
+      <h1 className="font-medium font-poppins text-[40px]">Profile</h1>
+      <div className="text-sm font-medium text-center text-gray-500 border-b border-gray-200 dark:text-gray-400 dark:border-gray-700">
+        <ul className="flex flex-wrap -mb-px">
+          <li className="mr-2">
+            <Link
```

Comment:
[nit] does this have to be a link or a button?
or did you want to make it add the navigation to the backstack?